.gitignore

```diff
@@ -1,5 +1,7 @@
 *.pyc
 *.swp
+build_branch_*/
```

Comment:
I do in source build, so I won't be affected by this, but this is probably really nice for those building this way (assuming they name the directory like this).